src/speed-index.js

```diff
@@ -2,6 +2,31 @@
 
 const imageSSIM = require('image-ssim');
 
+/* BEGIN FAST MODE CONSTANTS - See function doc for explanation */
+const FAST_MODE_ALLOWABLE_CHANGE_MAX = 5;
```

Comment:
speedline doesnt neccessarily follow the google js style guide.

i'd love to take that opportunity to not caps_lock our constants :)

You good with something like... `fastModeAllowableChangeMax` ?